src/core_ocean/shared/mpas_ocn_tidal_forcing.F

```diff
@@ -263,7 +266,8 @@ subroutine ocn_tidal_forcing_build_array(domain, meshPool, forcingPool, diagnost
         ! compute the tidalHeight
         if (trim(config_tidal_forcing_model) == 'monochromatic') then
           tidalHeight = config_tidal_forcing_monochromatic_amp * &
-            SIN(2.0_RKIND*pii/config_tidal_forcing_monochromatic_period * daysSinceStartOfSim) - &
+            SIN(2.0_RKIND*pii/config_tidal_forcing_monochromatic_period * daysSinceStartOfSim - &
```

Comment:
Nice!   And verified that default registry value for `config_tidal_forcing_monochromatic_phaseLag` is 0.